test/one_dragon/base/config/test_instance_force_login.py

```
@@ -0,0 +1,224 @@
+
+def _build_cfg(
+    monkeypatch: pytest.MonkeyPatch,
+    clients: dict[int, str],
+    active_idx: int,
+    active_in_od_indices: list[int],
+    instance_run: InstanceRun,
+) -> OneDragonConfig:
+    monkeypatch.setattr(od_config, 'GameAccountConfig', FakeGameAccountConfig)
+    FakeGameAccountConfig.clients = clients
+    monkeypatch.setattr(OneDragonConfig, 'instance_run', property(lambda self: instance_run.value.value))
+    # 用 __new__ 构建空白对象，跳过 YamlConfig.__init__，避免读取真实工作目录的 one_dragon 配置
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)